src/components/MetricsPane/charts/AggregateHardCoralCover.jsx

```diff
@@ -0,0 +1,89 @@
+import { useContext } from 'react'
+import Plot from 'react-plotly.js'
+
+import { ChartSubtitle, ChartWrapper, TitlesWrapper } from './Charts.styles'
+import { FilterProjectsContext } from '../../../context/FilterProjectsContext'
+import { MetricCardH3 } from '../MetricsPane.styles'
+import dashboardOnlyTheme from '../../../styles/dashboardOnlyTheme'
+
+const chartTheme = dashboardOnlyTheme.plotlyChart
+
+export const AggregateHardCoralCover = () => {
+  const { displayedSurveys } = useContext(FilterProjectsContext)
+
+  const hardCoralAveragesPerSurvey = displayedSurveys
+    .map((record) => {
+      const { protocols } = record
+
+      const benthicPitAverage =
+        protocols.benthicpit?.percent_cover_benthic_category_avg?.['Hard coral'] ?? 0
+      const benthicLitAverage =
+        protocols.benthiclit?.percent_cover_benthic_category_avg?.['Hard coral'] ?? 0
+      const benthicPqtAverage =
+        protocols.benthicpqt?.percent_cover_benthic_category_avg?.['Hard coral'] ?? 0
+      const quadratBenthicAverage = protocols.quadrat_benthic_percent?.percent_hard_avg_avg ?? 0
+
+      const numerator =
+        benthicLitAverage + benthicPitAverage + benthicPqtAverage + quadratBenthicAverage
+      const denominator =
+        (benthicPitAverage ? 1 : 0) +
+        (benthicLitAverage ? 1 : 0) +
+        (benthicPqtAverage ? 1 : 0) +
+        (quadratBenthicAverage ? 1 : 0)
+
+      return !denominator ? undefined : numerator / denominator
```

Comment:
This math is probably 'good enough' but there will be a discussion to review it later as it produces more of a gist or approximation of the combined average than a true average

Reply:
@saanobhaai ☝️